src/transformers/modeling_tf_bert.py

```diff
@@ -801,8 +829,31 @@ def __init__(self, config, *inputs, **kwargs):
         output_type=TFBaseModelOutputWithPooling,
         config_class=_CONFIG_FOR_DOC,
     )
-    def call(self, inputs, **kwargs):
-        outputs = self.bert(inputs, **kwargs)
+    def call(self,
+        inputs,
+        attention_mask=None,
+        token_type_ids=None,
+        position_ids=None,
+        head_mask=None,
+        inputs_embeds=None,
+        output_attentions=None,
+        output_hidden_states=None,
+        return_dict=None,
+        training=False
+    ):
+        outputs = self.bert(
+            inputs,
+            attention_mask,
```

Comment:
Can we use keyword arguments here? It's impossible to copy/paste and adapt for other models when we just feed the args, and less readable.